### PR TITLE
Use bundler version from the project to run bundle install

### PR DESCRIPTION
**Pull request by h13ronim**

### CODE REVIEW

File: bi.sh [hunk 1]
```diff
@@ -1,9 +1,11 @@
 unalias bi
 function bi() {
+  local bi_BUNDLER_VERSION_PARAM=""
```
Comment (pr0d1r2) [inline bi.sh]: Why is this initialized with `""`, this line seems redundant.

**Review by pr0d1r2 — CHANGES_REQUESTED** [review]
♻️  we can improve

File: bi.sh (hunk 1 above)
Comment (pr0d1r2) [reply bi.sh]: After all if we require `local` declaration of it we can just do:

```bash
local bi_BUNDLER_VERSION_PARAM
```

**Comment by h13ronim** [comment]
@pr0d1r2 check again :)

**Review by pr0d1r2 — APPROVED** [review]

**Pull request merged**